pkg/router/f5/types.go

```diff
@@ -106,6 +106,9 @@ type f5Policy struct {
 	// Name is the name of the policy.
 	Name string `json:"name"`
 
+	// Partition name for the policy
+	TmPartition string `json:"tmPartition"`
```

Comment:
Felt a little weird that this one is called `TmPartition` but every other F5 api call payload uses `Partition`. Thanks for explaining that one.